docs/MAIN_CONTROLLER_SPECS.md

```diff
@@ -0,0 +1,34 @@
+### Specifications
+
+|Feature||
+|:------|:-----|
+|Streaming Sources | Airplay, Pandora, Spotify Connect, DLNA  |
+|||
+|Control Interfaces| Web App, REST API|
+|||
+|Dimensions        | 2U 19" rackmount unit with 300mm depth|
+|||
+|Connectivity      | 100Mbps Ethernet|
+|                  | HDMI 1.4 output|
+|                  | 2x USB 2.0 External|
+|                  | 1x USB 2.0 Internal|
+|||
+|Power Input       | 90-132VAC or 180-264VAC|
+|||
+|Power Consumption | 6.7W Idle|
+|                  | 391W Continuous|
+|                  | 782W Peak|
+|||
+|Peak Current Consumption|6.8A @ 115VAC or 3.4A @230VAC|
+|||
+|Speaker Power     | 79 WPC @ 4 Ω|
+|                  | 57 WPC @ 4 Ω|
+|                  | 44 WPC @ 4 Ω|
+|||
+|Speaker Impedance | 4 -8 Ω |
+|||
+|Stereo Zones      | 6 |
+|||
+|Audio Input       | 4 RCA Line In|
+|||
+|Digital Audio Sources | 32-bit 384KHz (source 1), 16-bit 48KHz (sources 2-4)|
```

Comment:
Grab the latest specs from here: https://amplipro.com/wp-content/uploads/2023/12/amplipro_datasheet.pdf